doc/source/whatsnew/v0.18.2.txt

```diff
@@ -154,3 +154,4 @@ Bug Fixes
 - Bug in ``Period`` addition raises ``TypeError`` if ``Period`` is on right hand side (:issue:`13069`)
 - Bug in ``Peirod`` and ``Series`` or ``Index`` comparison raises ``TypeError`` (:issue:`13200`)
 - Bug in ``pd.set_eng_float_format()`` that would prevent NaN's from formatting (:issue:`11981`)
+- Bug in ``pd.read_csv()`` that prevents ``usecol`` kwarg from accepting unicode (:issue:`13219`)
```

Comment:
So we can't say "accepting unicode" in a broad sense, as there is limitation for 2bytes now.